index.html

```diff
@@ -14,46 +14,58 @@
 <body>
     <header id="header">
         <a href="#" class="logo">My Logo</a>
-        <nav>
+        <nav class="mobile-nav">
             <img src="./images/Enabled.png" alt="humburger menu">
         </nav>
+        <nav class="desk-nav">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
```

Comment:
- [ ]  Your desktop version is almost there, let's make it even more similar to the Figma design:

![Captura de pantalla 2022-03-26 a la(s) 11 10 12 a  m](https://user-images.githubusercontent.com/83717285/160248022-58092cf9-bf6f-4c6c-afc4-00c3b606bd03.png)


- Please make your header text wider and add more space around the logo and navigation links.
- It would be better to remove the gray background on the navigation bar which is not part of the template.